src/Schematron.Tests/ValidatorTests.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.IO;
+using System.Xml;
+using System.Xml.XPath;
+using System.Xml.Serialization;
+using System.Text;
+using System.Xml.Schema;
+using Xunit;
+
+namespace Schematron.Tests
+{    
+    public class ValidatorTests
+    {
+        const string XsdLocation = "./Content/po-schema.xsd";
+        const string XsdWithPartialSchemaLocation = "./Content/po-schema-with-schema-import.xsd";
+        const string XmlContentLocation = "./Content/po-instance.xml";
+        const string TargetNamespace = "http://example.com/po-schematron";
+
+        [Fact]
+        public void NewAddSchemaSignatureShouldNotBreakCode()
```

Comment:
This test is weird, it has no verification/assert? Maybe it should Assert.Throws<ValidationException> instead?